File: track/tests/factories.py
import factory
from track import models


class TrackFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = models.Track

    gateway_id = factory.Faker('pyint')
    title = factory.Faker('sentence', nb_words=3)
    stream_url = factory.Faker('uri')
    artwork_url = factory.Faker('uri')
    username = factory.Faker('name')
    archived = False

File: assets/src/js/tests/factories.js
import faker from 'faker'

export function trackFactory() {
  return {
    artwork_url: faker.internet.url(),
    gateway_id: faker.random.number(),
    id: faker.random.number(),
    stream_url: faker.internet.url(),
    title: faker.lorem.sentence(),
    username: faker.internet.userName(),
  }
}

